src/ai/defaultai.cc

```diff
@@ -2060,11 +2075,11 @@ void DefaultAI::update_buildable_field(BuildableField& field) {
 		   management_data.neuron_pool[10].get_result_safe(field.military_loneliness / 50, kAbsValue);
 
 		score_parts[30] =
-		   -10 * management_data.neuron_pool[8].get_result_safe(
-		            3 * (field.military_in_constr_nearby + field.military_unstationed), kAbsValue);
-		score_parts[31] =
-		   -10 * management_data.neuron_pool[31].get_result_safe(
+		   -20 * management_data.neuron_pool[8].get_result_safe(
 		            3 * (field.military_in_constr_nearby + field.military_unstationed), kAbsValue);
+		// score_parts[31] =
```

Comment:
this should be reinstated to the former state as we won't need the neuron no 31 for the calculation in the following

Reply:
I did this to release one magic number, and not 31, but other, it is not visible in the diff here.. but I think I used it elsewhere

Reply:
you released neuron number 31 which is used now for malus calculation. However there are a lot of neurons still unused (e.g. all neurons from 35 to 41, 68 to 72 and many more so no need to free any.
Unrelated Issue is that some of them are used multiple times which should be changed to give the AI better possibilities to train. (multiple used neurons are 4, 8, 18, 23(4x))

Reply:
OK, reverted it. I will try to do some housekeeping with these neurons